pretrained.md

```diff
@@ -39,3 +39,5 @@ The pretrained embeddings are trained using
 | [French](http://www.sfs.uni-tuebingen.de/a3-public-data/finalfusion/conll2017/french-skipgram-mincount-30-ctx-10-dims-300-2020-06-30.fifu)      | skipgram (2020-06-30)| [CoNLL 2017 raw](https://lindat.mff.cuni.cz/repository/xmlui/handle/11234/1-1989)    | [Metadata](http://www.sfs.uni-tuebingen.de/a3-public-data/finalfusion/conll2017/french-skipgram-mincount-30-ctx-10-dims-300-2020-06-30.txt)       |
 | [Galician](http://www.sfs.uni-tuebingen.de/a3-public-data/finalfusion/conll2017/galician-structgram-mincount-30-ctx-10-dims-300-2020-07-01.fifu)      | structured skipgram (2020-07-01)| [CoNLL 2017 raw](https://lindat.mff.cuni.cz/repository/xmlui/handle/11234/1-1989)    | [Metadata](http://www.sfs.uni-tuebingen.de/a3-public-data/finalfusion/conll2017/galician-structgram-mincount-30-ctx-10-dims-300-2020-07-01.txt)       |
 | [Galician](http://www.sfs.uni-tuebingen.de/a3-public-data/finalfusion/conll2017/galician-skipgram-mincount-30-ctx-10-dims-300-2020-06-30.fifu)      | skipgram (2020-06-30)| [CoNLL 2017 raw](https://lindat.mff.cuni.cz/repository/xmlui/handle/11234/1-1989)    | [Metadata](http://www.sfs.uni-tuebingen.de/a3-public-data/finalfusion/conll2017/galician-skipgram-mincount-30-ctx-10-dims-300-2020-06-30.txt)       |
+
+<a rel="license" href="http://creativecommons.org/licenses/by/2.0/"><img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by/2.0/88x31.png" /></a><br />These embeddings are licensed under a <a rel="license" href="http://creativecommons.org/licenses/by/2.0/">Creative Commons Attribution 2.0 Generic License</a>.
```

Comment:
Seems like an oldie?

```suggestion
<a rel="license" href="http://creativecommons.org/licenses/by/4.0/"><img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by/4.0/88x31.png" /></a><br />This work is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution 4.0 International License</a>.
```

Reply:
Uhm, yeah, I just googled CC-BY and went with the first hit, oops. wonder what happens if I use the `Commit suggestion` button here!

Reply:
Meh, that adds another commit